src/main/java/bio/terra/service/dataset/DatasetDao.java

```diff
@@ -62,10 +63,27 @@
         " dataset.id, name, description, default_profile_id, project_resource_id, application_resource_id, " +
             "created_date, ";
 
-    private static final String datasetStorageQuery = "(SELECT jsonb_agg(sr) " +
-            "FROM (SELECT region, cloud_resource as \"cloudResource\", " +
-            "lower(cloud_platform) as \"cloudPlatform\", dataset_id as \"datasetId\" " +
-            "FROM storage_resource WHERE dataset_id = dataset.id) sr) AS storage ";
+    private static final String datasetStorageQuery = "(SELECT jsonb_agg(json_build_object( " +
```

Comment:
We don't yet do this everywhere but I think these should be uppercase under java code conventions.
```suggestion
    private static final String DATASET_STORAGE_QUERY = "(SELECT jsonb_agg(json_build_object( " +
```